_includes/event-post-macros.html

```diff
@@ -0,0 +1,226 @@
+          <span class="event-meta_time">{{ post.date | date('%H:%M %p %Z') }}</span>
+        </time>
+    </div>
+{% endmacro %}
+
+
+{# ==========================================================================
+
+   event_meta_datetime()
+
+   ==========================================================================
+
+   Description:
+
+   Create HTML event meta date/time markup when given:
+
+   date: A post date from a query result.
+
+   ========================================================================== #}
+
+{% macro event_meta_datetime(date) %}
+    <time class="event-meta_datetime dt-start" datetime="{{ date | date('%Y-%m-%d %H:%M %p %Z') }}">
+      <span class="event-meta_date">{{ date | date('%m/%d/%Y') }}</span>
+      <span class="event-meta_time">{{ date | date('%H:%M %p %Z') }}</span>
+    </time>
+{% endmacro %}
+
+
+{# ==========================================================================
+
+   post_summary()
+
+   ==========================================================================
+
+   Description:
+
+   Create an event post summary when given:
+
+   post: A post from a query result.
+
+   path: The path to which post filters are applies.
+
+   ========================================================================== #}
+
+
+{% macro post_summary(post, path) %}
+
+{%- import "tags.html" as tags %}
+    <a href="{{ post.permalink }}">
+        <header class="summary_header">
+            {{ calendar_icon(post.date) }}
+                <div class="summary-meta_container">
+                    <h1 class="summary_heading">{{ post.title|safe }}</h1>
+                    {{ event_meta(post) }}
+                </div>
+        </header>
+        <p class="summary_text">
+            {% if post.dek %}
+                {{ post.dek|striptags }}
+            {% else %}
+                {{ post.excerpt|striptags }}
+            {% endif %}
+        </p>
+    </a>
+    <footer>
+      {{ tags.render(post.tags, path) }}
+    </footer>
+{% endmacro %}
+
+
+
+{# ==========================================================================
+
+   event_venue()
+
+   ==========================================================================
+
+   Description:
+
+   Create an event post venue when given:
+
+   post: A post from a query result.
+
+   path: The path to which post filters are applies.
+
+   ========================================================================== #}
+
+
+{% macro event_venue(post) %}
+    {%- set city =  (post.location or 'Washington, DC') -%}    
+    {%- set street = (post.street or '1275 First St NE') -%}
+    {%- set venue = (post.venue or 'Verizon Center') -%}
+    {%- set zip = (post.zip_code or '20002') -%}
+
+    <section class="event-venue">
+        <header class="event-venue_header">
+            <h2 class="event-venue_heading">Washington, DC</h2>
+            <div class="content-l">
+                <div class="content-l_col content-l_col-1-2">
```

Comment:
This should include `event-meta` I think? Looks like `event-meta_address` appears below, but it's not enclosed in the rest of the `event-meta` pattern.